README.md

```diff
@@ -1,42 +1,50 @@
 # Ember-Impagination
 [![Build Status](https://travis-ci.org/thefrontside/ember-impagination.svg)](https://travis-ci.org/thefrontside/ember-impagination)
 
-[Impagination](https://github.com/flexyford/impagination) implementation for Ember.
+*Ember-Impagination* is an Ember port of [Impagination](https://github.com/flexyford/impagination), a front-end data-layer for the paginated API on your server. *Ember-Impagination* leverages the power of Glimmer and provides your component the data it needs to render quickly each and every time. 
 > Whatever your use-case: infinite scrolling lists, a carousel browser, or even a classic page-by-page result list, Impagination frees you to focus on what you want to do with your data, not the micro-logistics of when to fetch it. All you provide Impagination is the logic to fetch a single page, plus how many pages you want it to pre-fetch ahead of you, and it will figure out the rest.
 
 > Impagination is built using an event-driven immutable style, so it is ideal for use with UI frameworks like Ember . . .
 
-Hence, we present *ember-impagination*
+Hence, we present *Ember-Impagination*
```

Comment:
how would you use `impagination` directly with the component? Isn't the only option to provide the attributes, and nothing else?